lib/components/header.js

```diff
@@ -109,30 +108,42 @@ export default class Header extends PureComponent {
 
     return (
       <header
-        className={css('header', isMac && 'headerRounded')}
+        className={`header_header ${isMac && 'header_headerRounded'}`}
```

Comment:
have you checked the result of this? when `isMac` is falsy it should render `header_header false`

Reply:
you might want to use a ternary

Reply:
or https://github.com/JedWatson/classnames which helps a lot, and makes it cleaner for when you have more than one dynamic class name 👍 Also helps with consistency

Reply:
`header_header` sound like a bad name. You are already inside `Header` component. Since you are at the root className of the component. You could leave it `header` and then `header_rounded`

Reply:
Thanks for spotting that @giuseppeg. I changed to a ternary in a few places but I obviously missed a lot somehow.

@ppot I didn't make those names up, that's the names that we were using before but they were just generated from `base_components`. I did speak about that in the PR description.
Changing these classnames would break users custom CSS, themes, and some plugins.

Reply:
Yeah. That's why my comment at the end: `My only concern is with the names ` haha  ;)

Reply:
It is something we need to revisit for the codebase. But for now it's okay to leave it like that.